docs/docs/gatsby-image-api.md

```diff
@@ -0,0 +1,344 @@
+---
+title: Gatsby Image API
+---
+
+Part of what makes Gatsby sites so fast is its recommended approach to handling images. `gatsby-image` is a React component designed to work seamlessly with Gatsby’s [native image processing](https://image-processing.gatsbyjs.org/) capabilities powered by GraphQL and [gatsby-plugin-sharp](/packages/gatsby-plugin-sharp/) to easily and completely optimize image loading for your sites.
+
+> _Warning: gatsby-image is **not** a drop-in replacement for `<img />`. It’s optimized for fixed width/height images and images that stretch the full-width of a container._
```

Comment:
"Warning" makes it sound like it might be scary — swapping it out for a note makes it more of a heads-up.
```suggestion
> _Note: gatsby-image is **not** a drop-in replacement for `<img />`. It’s optimized for fixed width/height images and images that stretch the full-width of a container._
```